Sources/SwiftFormatRules/ValidateDocumentationComments.swift

```diff
@@ -153,4 +171,15 @@ extension Diagnostic.Message {
     "replace the singular inline form of 'Parameter' tag with a plural 'Parameters' tag "
       + "and group each parameter as a nested list"
   )
+
+  public static func removeThrowsComment(funcName: String) -> Diagnostic.Message {
+    return Diagnostic.Message(
+      .warning,
+      "remove the throws comment of \(funcName), it doesn't throw an error"
```

Comment:
Suggested message: `"remove the 'Throws' tag for non-throwing function \(funcName)"`

(I realize that yours is similar to the `return` message, but it should be improved at some point as well; we can do that in a separate change.)

Reply:
Fixed.